load-test/scenarios/teacher-generate-pins/teacher-generate-pins.jmx

```diff
@@ -31,10 +31,24 @@
         <boolProp name="CookieManager.clearEachIteration">false</boolProp>
       </CookieManager>
       <hashTree/>
+      <Arguments guiclass="ArgumentsPanel" testclass="Arguments" testname="User Defined Variables" enabled="true">
+        <collectionProp name="Arguments.arguments">
+          <elementProp name="TEACHER_LOGIN_FILE" elementType="Argument">
+            <stringProp name="Argument.name">TEACHER_LOGIN_FILE</stringProp>
+            <stringProp name="Argument.value">${__BeanShell( System.getenv(&quot;FLOOD_NODE_INDEX&quot;) + &quot;-teacherLogins.csv&quot; )}</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+        </collectionProp>
+      </Arguments>
+      <hashTree/>
       <CSVDataSet guiclass="TestBeanGUI" testclass="CSVDataSet" testname="Teacher Logins DataSet" enabled="true">
         <stringProp name="delimiter">,</stringProp>
         <stringProp name="fileEncoding"></stringProp>
+<<<<<<< HEAD:load-test/scenarios/teacher-generate-pins/teacher-generate-pins.jmx
```

Comment:
@GuyHarwood seems like this is an incorrect merge from a conflict?

Reply:
good spot, fixed 👍